core/src/main/java/org/apache/accumulo/core/dataImpl/KeyExtent.java

```diff
@@ -495,11 +495,12 @@ public Range toDataRange() {
   }
 
   public Range toMetadataRange() {
-    Text metadataPrevRow = new Text(getTableId().getUtf8());
-    metadataPrevRow.append(new byte[] {';'}, 0, 1);
-    if (getPrevEndRow() != null) {
-      metadataPrevRow.append(getPrevEndRow().getBytes(), 0, getPrevEndRow().getLength());
-    }
+    Text metadataPrevRow;
+
+    if (textPrevEndRow == null)
```

Comment:
I would suggest calling getPrevEndRow() here as it makes it more consistent with the rest of the function.